IBAnimatableApp/TransitionTableViewController.swift

```diff
@@ -45,46 +45,185 @@ class TransitionTableViewController: UITableViewController {
 private extension TransitionTableViewController {
 
   func populateTransitionTypeData() {
+
     transitionAnimationsHeaders.append("Fade")
-    transitionAnimations.append(["Fade", "Fade(In)", "Fade(Out)"])
+
+    let fadeAnimations: [TransitionAnimationType] = [.fade(direction: .in), .fade(direction: .out), .fade(direction: .cross)]
+
+    transitionAnimations.append(toString(animations: fadeAnimations))
+
     transitionAnimationsHeaders.append("SystemCube")
-    transitionAnimations.append(transitionTypeWithDirections(forName: "SystemCube"))
+    let cubeAnimations: [TransitionAnimationType] = [
+      .systemCube(from: .left), .systemCube(from: .right), .systemCube(from: .top), .systemCube(from: .bottom)]
+
+    transitionAnimations.append(toString(animations: cubeAnimations))
+
     transitionAnimationsHeaders.append("SystemFlip")
-    transitionAnimations.append(transitionTypeWithDirections(forName: "SystemFlip"))
+    let flipSystemAnimations: [TransitionAnimationType] = [
```

Comment:
My two cents again 😬 

```swift
let flipSystemAnimations: [TransitionAnimationType] = [.systemFlip(from: .left),
                                                               .systemFlip(from: .right),
                                                               .systemFlip(from: .top),
                                                               .systemFlip(from: .bottom)]
```

Can't get the indentation working on github, but the idea is to have just one aligned column for the parameters. Xcode indent it as expected